src/transform/ThemeTransform.js

```diff
@@ -80,13 +90,16 @@ const classifyElements = element => {
   /* en > Wikipedia:Graphs_and_charts > 801754530 */
   /* en > PepsiCo > 807406166 */
   /* en > Lua_(programming_language) > 809310207 */
+  /* en > Wikipedia:How_to_read_a_color_infobox > 762137635 */
   const selector = [
     'div.color_swatch div',
     'div[style*="position: absolute"]',
     'div.barbox table div[style*="background:"]',
     'div.chart div[style*="background-color"]',
     'div.chart ul li span[style*="background-color"]',
     'span.legend-color',
+    'table span[style*="background-color"]',
+    'table td[style*="background-color"]',
```

Comment:
We've got so many of these background attribute selectors now that I wonder if we can generalize them.

Reply:
Not sure.